build/workspace.bzl

```diff
@@ -104,16 +113,18 @@ def debian_image_dependencies():
             digest = _digest(_DEBIAN_BASE_DIGEST, arch),
             registry = "us.gcr.io/k8s-artifacts-prod/build-image",
             repository = "debian-base",
+            # Ensure the digests above are updated to match a new tag
             tag = "v2.1.0",  # ignored, but kept here for documentation
         )
 
         container_pull(
             name = "debian-iptables-" + arch,
             architecture = arch,
             digest = _digest(_DEBIAN_IPTABLES_DIGEST, arch),
-            registry = "k8s.gcr.io",
+            registry = "us.gcr.io/k8s-artifacts-prod/build-image",
```

Comment:
Is the bucket only available there or would `gcr.io/…` work too?

Reply:
@saschagrunert -- `gcr.io/...` would _NOT_ work.
`us.gcr.io` is one of the geo-specific endpoints images are promoted into (the others being `{eu,asia}.gcr.io).

Once the vanity domain flip (VDF) is complete (sometime in May), `k8s.gcr.io` will geo-target the appropriate endpoints.

We will then sweep references to `us.gcr.io/k8s-artifacts-prod/<staging-project>` and revert them to `k8s.gcr.io`.

More details: https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/Vanity-Domain-Flip.md
Tracking issue: https://github.com/kubernetes/release/issues/270

Reply:
That helps, thank you for the clarification 🙏